src/controller/CHIPDevice.cpp

```diff
@@ -622,6 +625,78 @@ void Device::OnSessionEstablished()
     }
 }
 
+void Device::ReleaseDAC()
+{
+    if (mDAC != nullptr)
+    {
+        Platform::MemoryFree(mDAC);
+    }
+    mDACLen = 0;
+    mDAC    = nullptr;
+}
+
+CHIP_ERROR Device::SetDAC(const ByteSpan & dac)
```

Comment:
This seems like it doesn't belong on the Device object, but rather on the Commissioner, in a separate data structure indexed by device. Otherwise all Matter devices (including lightbulbs) that need to support node-to-node communication, but do not necessarily need to do device attestation, will carry this code.

@pan-apple Do you have a suggestion how to resole?